service/pkg/util/fqn.go

```diff
@@ -11,15 +11,25 @@ type FullyQualifiedResourceMappingGroup struct {
 	GroupName string
 }
 
+type FullyQualifiedRegisteredResourceValue struct {
+	Fqn   string
+	Name  string
+	Value string
+}
+
 // protovalidate already validates the FQN format in the service request
```

Comment:
This is actually only true for the Resource Mapping Group FQNs, as all other attribute/value FQNs only validate that a proper URI is provided. Should we only care that a proper URI is provided in the request protos?

Reply:
Per discussion with @jakedoublev, we decided sticking to the protovalidate `string.uri` validation rather than individual FQN regexes is the best path forward. Then the service/db code can handle parsing them for accuracy.